node/service/src/chain_spec/frequency.rs

```diff
@@ -24,17 +24,32 @@ pub mod frequency_mainnet_keys {
 	pub const MAINNET_FRQ_SUDO: &str =
 		"0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d"; // Alice
 
-	//TODO: final collator key(s) for mainnet
-	pub const COLLATOR_1_SR25519: &str =
-		"0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d"; // Alice
-	pub const COLLATOR_2_SR25519: &str =
-		"0x8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48"; // Bob
-	pub const COLLATOR_3_SR25519: &str =
-		"0x90b5ab205c6974c9ea841be688864633dc9ca8a357843eeacf2314649965fe22"; // Charlie
-	pub const COLLATOR_4_SR25519: &str =
-		"0x306721211d5404bd9da88e0204360a1a9ab8b87c66c1bc2fcdd37f3c2222cc20"; // Dave
-	pub const COLLATOR_5_SR25519: &str =
-		"0xe659a7a1628cdd93febc04a4e0646ea20e9f5f0ce097d9a05290d4a9e054df4e"; // Eve
+	//TODO: update collator key naming convention
```

Comment:
I think this is fine. Perhaps move the session key next to the corresponding controller key, but otherwise good enough to produce the genesis. Then it is removed eventually anyway.

Reply:
I think I like that one